### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by omaflak**
Added a friendly maven central version badge

### CODE REVIEW

**Comment by omaflak** [comment]
I signed it!

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Glide
 =====
 
-[![Build Status](https://travis-ci.org/bumptech/glide.svg?branch=master)](https://travis-ci.org/bumptech/glide)
+[![Maven Central](https://maven-badges.herokuapp.com/maven-central/com.github.bumptech.glide/glide/badge.svg)](https://maven-badges.herokuapp.com/maven-central/com.github.bumptech.glide/glide/badge.svg) [![Build Status](https://travis-ci.org/bumptech/glide.svg?branch=master)](https://travis-ci.org/bumptech/glide)
```
Comment (TWiStErRob) [inline README.md]: shouldn't it link to the maven central instead of an SVG?

Comment (omaflak) [inline README.md]: It is linked to maven central, the link will provide an SVG of the current version of Glide.

Comment (TWiStErRob) [inline README.md]: Yes, but when a user clicks on it, it opens https://maven-badges.herokuapp.com/maven-central/com.github.bumptech.glide/glide/badge.svg which is pretty useless. It should open something like this: https://search.maven.org/#search%7Cgav%7C1%7Cg%3A%22com.github.bumptech.glide%22%20AND%20a%3A%22glide%22 not sure what other repos do.

Comment (omaflak) [inline README.md]: Oups you're right ! I'll change that!

**Comment by sjudd** [comment]
Thanks!

**Pull request merged**